packages/lb-components/src/components/pointCloudView/hooks/usePointCloudViews.ts

```diff
@@ -887,7 +901,14 @@ export const usePointCloudViews = () => {
       }
 
       const newPointCloudList = updateSelectedBox(newBoxParams);
-      syncPointCloudViews(fromView, newPolygon, newBoxParams, undefined, newPointCloudList);
+
+      syncPointCloudViews({
+        omitView: updateCurrentView ? undefined : fromView,
+        polygon: newPolygon,
+        boxParams: newBoxParams,
+        newPointCloudBoxList: [newBoxParams],
```

Comment:
This will trigger miss some propertities of boxParmas。
'newBoxParams' should get from value of updateSelectedBox(newBoxParams)